titan/params/outputs.yml

```diff
@@ -36,3 +36,21 @@ outputs:
       - drug_types
       - locations
       - components
+  logging:
+    to_file:
```

Comment:
Would it make more sense to put this as `stream` with file and stdout?

Reply:
`destination`?

Reply:
That sounds good to me!